website/pages/index.html

```diff
@@ -114,7 +114,8 @@ <h1 class="logo tex" data-expr="\KaTeX">KaTeX</h1>
     <div class="footer"><div class="container">
       <a class="ka-logo" href="https://www.khanacademy.org/"><img src="img/khan-academy.png" alt="Khan Academy" width="180" height="17"></a>
       <span class="credits">
-        Created by <a href="https://github.com/xymostech">Emily Eisenberg</a> and <a href="https://sophiebits.com/">Sophie Alpert</a>
+        Created by <a href="https://github.com/xymostech">Emily Eisenberg</a> and <a href="https://sophiebits.com/">Sophie Alpert</a><br>
+        Maintained by <a href="https://github.com/orgs/KaTeX/people">the KaTeX team</a> with help from <a href="https://github.com/Khan/KaTeX/graphs/contributors">contributors</a>
```

Comment:
Nice!

Reply:
@kevinbarabash This also includes the link to KaTeX organization. Should I remove it?

Reply:
I didn't notice that.  Yeah, let's remove that as well.

Reply:
@kevinbarabash Could you suggest how should I write it?

Reply:
What do you think of `KaTeX was built from the hard work of all these contributors.`

Reply:
Sounds good!  Slight tweak: Maybe `... hard work of [many contributors](https://github.com/Khan/KaTeX/graphs/contributors)`. I think it'd be good to keep the "created by" line as well.  Maybe `KaTeX was built from the hard work ...`  It was originally created by ...`?

Reply:
@edemaine I was just thinking to append the new line after existing line.